bip-0340.mediawiki

```diff
@@ -184,7 +184,7 @@ The algorithm ''Verify(pk, m, sig)'' is defined as:
 * Let ''s = int(sig[32:64])''; fail if ''s &ge; n''.
 * Let ''e = int(hash<sub>BIP0340/challenge</sub>(bytes(r) || bytes(P) || m)) mod n''.
 * Let ''R = s⋅G - e⋅P''.
-* Fail if ''not has_even_y(R)'' or ''x(R) &ne; r''.
+* Fail if ''is_infinite(R)'', or ''not has_even_y(R)'', or ''x(R) &ne; r''.
```

Comment:
Can we make this two "Fail" lines to avoid relying on the short-circuit evaluation of "or"?

Reply:
I made it three lines.